web_m2x_options/README.rst

```diff
@@ -127,6 +127,11 @@ Known issues
 Double check that you have no inherited view that remove ``options`` you set on a field ! 
 If nothing works, add a debugger in the first line of ``get_search_result method`` and enable debug mode in Odoo. When you write something in a many2one field, javascript debugger should pause. If not verify your installation.
 
+This module fixes an Odoo bug in the pop up of search that limit the items quantity
```

Comment:
Just let me remind you that addons is not the place to fix bugs. If you really consider this a bug, it should be fixed where it is: OCB.

Reply:
Indeed. but it should be done twice. If we fix Odoo, (via OCB) but install this module, this bug come back -;)

Reply:
Yes, it's just that adding this comment in the README is a little bit misleading. If there's a bug, then it shouldn't be documented in some random addon readme; it should be just fixed.

Also, if this addon avoids to introduce a bug, that's just a bug fix. No need to document that, unless you just want to add a note in the release notes.

I don't know if I explain myself :sweat_smile: 

Well in any case you got my approval here :blush: